### ISSUE TITLE
Time expiring during a multi-shot turn

### ISSUE THREAD

**Issue by hatj2** [etwa-rsc/rules#11]
This was originally phrased as follows, but is based on a misunderstanding of the rules. (See Patrick's email copied below for a better analysis.)

------------------------

The only definition of 'Turn' in the rules is in #18.1, and does not reflect current practice. 

> 18. After the timed period has ended, a round limit period is played. Play continues up to and including the turn of the colour that won the squidge-off, after which five further rounds of the colours are played, each round ending after the turn of the colour that won the squidge-off.
> 18.1 For the purposes of this rule, a turn is deemed to begin at the moment when the first shot in it is played. If the timed period expires between two shots of the same turn, it is deemed to have expired at the end of that turn.
> 18.2 If a colour cannot play a wink during a particular turn (owing to having no free winks or the forfeiture of a shot according to Rule 14), then this does constitute a turn when determining when the start of the five further rounds occurs and when counting how many rounds have been played.
> 

As we play it, it is your turn as soon as the previous player's last shot has finished - not as above. 

I think we should define what a turn is early on, and put it with the definition of a Shot.

**Comment by fluppeteer** [comment]
For my clarity: Red wins the squidge-off, green plays, red dithers before playing a shot, and time expires. By the rules, red ends 0. The question is whether red's dithering should be allowed to determine whether there should be more shots in round 0 - if red's turn started immediately after green's last shot, yellow, blue and green would also get to play, and red would play twice. Red can control this by deciding how long to take over a shot. This does feel mildly iffy. It radically affects what happens when squop-ups are involved.

I've certainly been rushing to get additional shots before rounds, and had opponents dither (maybe accidentally) only for time to expire. I've also stopped the clock on them with two seconds to go.

**Comment by hatj2** [comment]
How about something like this:

> N. A Turn is an opportunity for a player to play one or more shots, provided they have a free wink. 
N.1 A Turn may consist of: 
N.1.1 One or more shots. The player whose turn it is may play any one of their free winks, unless they are forfeiting a shot in accordance with #14. Should the player pot one of their winks they are awarded an extra shot within the same turn. 
N.1.1 A decision to pass. Any player may pass in lieu of taking a shot. 
N.1.2 An inability to play a shot, either by forfeiture of the shot by rule 14 or by having no free winks to play. 
N.2 A Turn ends: 
N.2.1 When all winks have come to rest on the field of play following the last shot in the turn; 
N.2.2 When the decision to Pass has been communicated to the opponents; 
N.2.3 When it has been determined that the player whose turn it is cannot make a shot. 
N.2.3.1 For example, in the case where an umpire is called to determine the status of the last wink of a particular colour on that colour's turn and determines that that wink is squopped, that turn comes to an end upon the umpire's pronouncement. 
N.2.3.2 For example, Green's Turn ends, and Red is forfeiting a shot under rule 14; Red's Turn is deemed to end immediately such that Yellow's turn begins when Green's Turn ends. 
N.3 The Colours take turns following each other in alphabetical order in English, such that Green follows Blue, Red follows Green, Yellow follows Red and Blue follows Yellow.

**Comment by fluppeteer** [comment]
Oh yes, I remember the nonsense. By strict reading of the rules, if currently a turn doesn't start until a shot is played, the colour playing the shot could choose to stop the clock because they were dithering, since it would technically be the turn of the opponents which was taking too long.

I think I like this, except the introduction ("N" itself), which I think confuses matters by bringing up opportunities to play and free winks but not going off (all clarified in subsections). 

I don't think N.1.1 needs to go into detail about a shot, since this is already covered in rules 9-12. Non-normative guidance is useful, but we need to tread carefully when it comes to things like potting two winks, passing once, then playing once (so the "turn" isn't a pass, the pass is a "shot" - we might just want to define it as such and fix the "shot" definition).

Even with this rule, a turn doesn't end until all winks that left the playing volume have been found. The team who are to play next could choose whether to stop the clock by finding such a wink and holding it hostage such that the previous turn continued. That comes under the "don't play silly buggers" category, but still...

This reminds me about the colour/Colour proposal. I'll raise that separately.

**Comment by fluppeteer** [comment]
I hope Harley can chime in on this, because we discussed it a bit. I think life is easier if we define a "shot" to include "being unable to play due to not having a wink", "being unable to play due to going off" and "passing". Harley also pointed out that defining a "flat wink" would make some wording easier. As an aside, I don't think the rules become less clear if the word "squopped" is replaced with "covered", and this appears to make translation much more trivial.

**Comment by hatj2** [comment]
> During a shot, a player exerts downward pressure of squidger on wink.
Not if 'passing' counts as a shot; not if forfeiting your shot counts as a shot (how can this possibly work, logically?). I strongly feel that the word 'shot' refers to actually touching a squidger to a wink or winks. 
If you say the word 'pass' and it isn't your turn, is that an illegal shot, by your definition?

**Comment by timhunt** [comment]
I agree it makes sense to define turn, and Harley's draf is pretty good.

It is a little bit sad if defining at turn takes that many words, and having done so, it does not shorten the rules anywhere else. Is there no scope for tidying up the free turns rules, for examples.

Given Harley's draft, I think it does not help to try to define passing as a shot. I think Harley's rule N.1, defining a turn as either one or more shots, or the inability or choice not to play a shot.

Is it sufficiently clear that you can pot the last free wink of your colour, or pot a wink and then choose to pass? Given this proposed chage, is it the thing about what happens if you pot a wink and go off in the same shot in the optimal place in the revised rules?

**Comment by fluppeteer** [comment]
I guess I'd be more comfortable if there was some "thing" that happened some number of times in a turn. I agree that "shot" is unfortunate, because we'd really like to have a term for playing a wink with a squidger. "Action" isn't much better, since being unable to play isn't really an "action". I do think Harley's version doesn't clearly transmit to the terminally confused that in one turn, one can play a shot that pots four winks and also sends a wink off, lose one of the four additional shots by having gone off, pass, play a shot that subs, and lose the last shot through not having any free winks. It feels to me as though the consequence of potting should be to have another... thing, where the thing is defined as one of the options.

As for "passing" when it's not your turn... the ambiguity came about, I think, because one team assuming that the other knew what was going on (or didn't) caused problems. Funny that. If there's a squop-up by Yellow/Green, it's Yellow's turn and Green says "pass", does that mean Green is randomly saying "pass" and nothing to do with the game, Green is speaking on behalf of Yellow, or Green is decreeing that Yellow has silently passed, and Green is passing as Green? What if it's singles and Green/Yellow are the same person? If it's Yellow's turn, the player who is both Green and Yellow says "pass" and then plays with Yellow, what happened?

**Comment by timhunt** [comment]
Email from Patrick Barrie setting out the real issue:

Dear Rules Sub-committee (Andrew/Ed/Tim/Harley/Larry)

I know that you have been considering the wording of rule 18/18.1, and I agree that it should be reworded. The current wording is effectively identical to that when I started winking career in 1984 so it has been around a long time. Consider 18.1 first.

“18.1 For the purposes of this rule, a turn is deemed to begin at the moment when the first shot in it is played. If the timed period expires between two shots of the same turn, it is deemed to have expired at the end of that turn.”

There are two separate sentences in 18.1 to consider.

Sentence 1. 

Taken in isolation, this suggests a turn begins when the first shot in it is played – which is not the intention. However, the key phrase is “for the purposes of this (specific) rule” – so 18.1 is not actually incorrect but it is potentially misleading. Elsewhere in the rules, it is clear that green’s turn starts immediately after blue’s turn comes to an end (rule 7.1), that a turn consists of a sequence of separate shots if winks of the correct colour are potted (rule 12.1), and that a shot ends when all winks have come to rest (rule 9.2).

You might think that deleting the first sentence of 18.1 removes any confusion but there is a problem. Consider blue being squidge-off winner. If it is blue to play when time elapses then blue’s turn has started even though no blue shot has been played. It is not clear from Rule 18 wording on its own that we would like this blue turn to end round zero – this is why the first sentence of 18.1 is there.

Sentence 2.

This addresses the situation that time elapses in the middle of a turn that consists of more than one shot because of potting. I admit that I think this rule (and current practice) is illogical – I think if time elapses during blue’s turn it should be treated consistently regardless of whether blue has already potted a wink in the turn or not. 

Possible changes.

Option (a). As a minimum, the two sentences in 18.1 should be separated because they are addressing two different situations. The first sentence of 18.1 could be appended to rule 18. Or the two sentences could become Rules 18.1 and 18.2, with current 18.2 become 18.3. With this option, there is no change to current practice. But some may argue 18.1 is misleading.

Option (b). Rewrite Rule 18 and 18.1. One possibility is:

“18. After the timed period has ended, a round limit period is played. Play continues using the normal sequence of colours. After the turn ends of the colour that won the squidge-off, five further rounds are played, with each round consisting of one turn for each colour in the normal sequence.” 
This wording is intended to make the rules clearer, and more comprehensible to beginners.

(i) it defines “rounds” far better than the current rules;

(ii) both sentences of current Rule 18.1 could be deleted;

(iii) but it is a change in current practice on what to do when time elapses in the middle of a turn with potting.  Some may not like this.

Best wishes,

Patrick

**Comment by timhunt** [comment]
I favour Patrick's option b). It is a change to current practice, but it makes the rules simpler, and will help tournaments run on time better, without being unfair to anyone.

**Comment by hatj2** [comment]
I think Patrick (and Andrew in his example above) haven't understood my problem with this. 

Suppose Blue won the squidge-off, and after Blue's last shot but before Green's first shot the time runs out. Under 18.1 it is still the turn of the player that won the squidge-off (Blue), so once that turn ends (by Green playing a shot) it is round 1. 

> Elsewhere in the rules, it is clear that green’s turn starts immediately after blue’s turn comes to an end (rule 7.1), that a turn consists of a sequence of separate shots if winks of the correct colour are potted (rule 12.1), and that a shot ends when all winks have come to rest (rule 9.2).

But the start/end points of a turn are not defined anywhere else. I still think we need to define turn elsewhere and reword 18.

**Comment by fluppeteer** [comment]
Sorry for the silence - I've picked up a weird ailment on the flight back to the UK, and am still a bit woozy. Perfect condition to try to make sense of rules. Email may take longer. I'm coming to this a bit rusty and confused.

12.1 defines a turn as a sequence of shots. 9.2 defines when a shot ends, but 9.1 defines when a shot starts, which is in conflict with 7.1. The issue appears to be that there are two concepts of "shot":

- the act of playing of a wink, resulting in its movement, which begins when a wink starts to move irreversibly and ends when the winks stop moving (etc.)
- the opportunity to play a shot, which begins once the last shot by the former definition ends

Both are part of rule 11 (passing instead of playing a shot and the pass constituting a shot). A turn is clearly a sequence of the latter.

By consistency with - and not necessarily advocating - the capitalisation terminology for colours, we can say a "shot" is the act of playing a wink, and a Shot is part of a turn (or possibly Turn). Although I'd be very happy to have some other terminology. I'm also using Player as a synonym for Colour (not necessarily the same as "colour" in a failure to free or "player" in a singles' game; pick your poison).

Here's another go at defining what Harley was going for in those terms:

N. The game comprises a sequence of Shots.
N.1 A Shot comprises one of:
N.1.1 The playing of a shot by the Player whose turn it is.
N.1.1 In this case, the Shot ends once all winks have come to rest on the field of play following the shot.
N.1.2 The Player whose Turn it is communicating the decision to pass to the opponents, in lieu of playing a shot.
N.1.2.1 In this case the Shot ends once this communication has been made.
N.1.3 The determination of the inability to play a shot by rule N+1.3 or by having no free winks to play.
N.1.3.1 In this case the Shot ends once this determination has been made.
N.1.3.2 For example, in the case where an umpire is called to determine the status of the last wink of a particular Colour on that Colour's turn and determines that that wink is squopped, that Shot comes to an end upon the umpire's pronouncement.
N.2 A Shot begins as soon as a previous Shot has ended.
N.2.1 The first Shot of the game begins with the start of the first shot; this also begins the time period of the game.
N+1. A Turn comprises a sequence of Shots.
N+1.1 The Colours take Turns following each other in alphabetical order in English, such that Green follows Blue, Red follows Green, Yellow follows Red and Blue follows Yellow.
N+1.2 If any Player whose turn it is pots any wink of that Player's Colour, the Player is awarded an extra Shot in the same Turn.
N+1.2.1 This award is cumulative if more than one wink is potted in a single shot.
N+1.3 If any Player whose turn it is causes any wink of that Player's Colour wholly or partially to leave the playing volume, the next Shot to be played by that Colour is forfeited by rule N.1.3.
N+1.3.1 This penalty is not cumulative; if more than one wink of the Player's Colour wholly or partially leaves the playing volume, only a single Shot is forfeited.
(Insert elaboration from rule 14)

Right, discussions have moved on, so let me post this much, then chime in on the rest.

**Comment by fluppeteer** [comment]
As discussed by email (incompetently, in my case), I think our options are as follows. Values in the table show the round in which the shot after time expires is played.

Edit: Added (in parentheses) which colour can control which round follows time expiring by dithering either side of time expiring, and slight paraphrase of some wording.

<table>
<tr>
<th>Rule</th>
<th>B<br />TE<br />B</th>
<th>Y<br />TE<br />B</th>
<th>G<br />TE<br />G</th>
<th>B<br />TE<br />G</th>
</tr>
<tr>
<td>1. "After time expires, play continues until the SoW completes a full Turn."</td>
<td>-1<br />(-)</td>
<td>-1<br />(Y)</td>
<td>0<br />(-)</td>
<td>0<br />(-)</td>
</tr>
<tr>
<td>2. "Play continues until the SoW completes a Turn in which no action has previously been taken."</td>
<td>-1<br />(B)</td>
<td>0<br />(Y/B)</td>
<td>0<br />(-)</td>
<td>0<br />(-)</td>
</tr>
<tr>
<td>3. "Play continues until the first end of a SoW Turn after time has expired." aka "The beginning of the first Turn of the CFSoW after time has expired begins round 1."</td>
<td>0<br />(B)</td>
<td>0<br />(B)</td>
<td>0<br />(-)</td>
<td>0<br />(B)</td>
</tr>
<tr>
<td>4. "Play continues until the end of the first Turn of the CFSoW in which no action has previously been taken, which then ends the first Turn of round 1."</td>
<td>0<br />(-)</td>
<td>0<br />(B)</td>
<td>0<br />(G)</td>
<td>1<br />(B/G)</td>
</tr>
<tr>
<td>5. "The first end of a CFSoW Turn to occur after time has expired ends the first Turn of round 1."</td>
<td>0<br />(-)</td>
<td>0<br />(-)</td>
<td>1<br />(G)</td>
<td>1<br />(G)</td>
</tr>
</table>

(In case 2, Yellow dithering can force Blue into ending round 0 with its next shot; Blue can avoid doing so by rushing. In case 4, Blue can force Green into round 1 by dithering, and Green can rush an action to stay in round 0 without ending Green's Turn.)

Assumes Blue won the squidge-off, "SoW" = "squidge-off winner" and "CFSoW" is "the Colour following the squidge-off winner". I'm using my definition of "Turn" - ending when the last shot in the turn ends, and beginning immediately after the previous Turn. "Action" here means "shot, declaration of pass, or determination of inability to play" as distinct from the "thinking time" component of my general definition of a "Shot".

For the cases, letters indicate a shot by the Colour beginning with that letter, "TE" indicates the point at which time expires".

I believe option 2 is the current rules. I believe Patrick proposed 3. I think 5 most embodies the spirit of "everyone gets to play in five rounds at the end of the game", but I don't mind 3 if we start saying "up to six turns" rather than "at least five". Option 3 means the squidge-off winner acts to take us into round 1; it's the only case where the SoW uniquely decides when rounds happen relative to time expiring, for better or for worse (if I've got it right, which I may not have).

**Comment by fluppeteer** [comment]
For archival reasons, Patrick quoth:

<hr>

Dear Rules Sub-Committee,

Thank you for responses which I have read carefully.

+++

1. I’m posting to e-mail rather than github because:

(i) I’m not sure how to post to github - feel free to upload my message 
there;

(ii) I’m not a member of rules sub-committee;

(iii) I want this e-mail message to constitute a formal written 
communication with the Chair of the Rules Sub-Committee (whoever that 
might be);

(iv) I’m old enough to remember two previous electronic forums that we 
have used for archiving discussions on possible changes to rules.

+++

2. I should say where I am coming from. On rules issues these days, I 
try to concentrate only on practical aspects: I am only really 
interested in wording changes that provide a clear improvement on what 
is currently written down. Any other changes proposed – e.g. to the way 
the game is played, or to address particular pedantic points unless the 
new wording is far clearer to players than the old – are likely to be 
rejected.

The specific proposal under discussion is to amend rule 18 to read the 
following, with deletion of rule 18.1.

“18. After the timed period has ended, a round limit period is played. 
Play continues using the normal sequence of colours. After the turn ends 
of the colour that won the squidge-off, five further rounds are played, 
with each round consisting of one turn for each colour in the normal 
sequence.”

+++

3. I accept Harley’s point that the rules do not explicitly define when 
a turn ends. However, I do not think this is actually a problem. The 
concept of “turn” is well established in the English language, and it is 
familiar to players of board games (and a few sports). I also accept 
Andrew’s point that there is a real distinction between what he terms 
“shot” and “Shot”. However, I’m not convinced that the majority of 
players would find this distinction helpful, even if we were 
old-fashioned and called them “squidge” and “shot” instead.

3(a) It is quite hard to put in a definition of a turn in an elegant 
fashion within the current rules structure. Any inelegant proposal is 
likely to be rejected. [Aside: in 2003-4 I did consider having a 
separate rules section on “definitions”, but this proved hard to 
implement in a useful way and so kept the “old” system of having 
definitions at relevant scattered points in the rules but now with a 
glossary of terms at the end.]

3(b) I don’t think examples of the start of the “five further rounds” 
situations are required as a sub-rule – the proposed new wording is 
clear enough – but examples could be included in the “notes and 
guidance” section if there was the will to do so.

+++

4. I accept Andrew’s point that the number of rounds played after the 
timer elapses varies – we don’t play 5 rounds; we play 5 rounds plus a 
number of turns. I am content with this. I don’t think anyone wants to 
change how rounds are currently counted - indeed, I think any proposal 
to change how rounds are currently counted is doomed to failure even if 
it was logical. [Aside: the current proposal is an exception because the 
number of games in which time elapses immediately after the squidge-off 
winner has potted a wink is very small and many players don’t know what 
the current rule prescribes in this situation anyway.]

+++

5. I accept Larry’s point that my proposed wording for time elapsing 
during a squidge-off winner’s turn may change play slightly in one-off 
situations – but I don’t think this is significant because it is already 
the case that players hurry up/slow down just before time in order to 
get/prevent an extra round of winks.

+++

6. Andrew has responded to Larry’s point on Rule 17. The timed limit 
period is exactly 25 minutes for pairs games – but subject to the 
“stopping the clock” situations listed. The round limit period starts 
immediately after the timer goes “bing”. The round limit period contains 
at least five further rounds, not exactly five further rounds.

+++

7. I would be grateful if the Rules Sub-Committee could deliberate on 
whether they wish to propose the wording I have suggested. If they do 
not, then I have the possibility of proposing it myself as a “private 
member’s bill” by notifying the ETwA Secretary. I can do this because I 
have submitted my proposal in writing for prior consideration of the 
Rules Sub-Committee; at a formal rules meeting, the Rules Sub-Committee 
have the option of supporting or opposing my motion.

+++

8. There is one other aspect of the counting of rounds that I find 
bizarre, and that is what is prescribed when time elapses during a 
squop-up (rule 22.3.4), with the squopped-up pair allowed to play a turn 
“in round zero”. There can be lots of extra turns before the 5 and a bit 
rounds start. Everyone finds this rule strange the first time they 
encounter it. Personally I would far rather 22.3.4 read similar to 
22.3.3 with normal counting of rounds in all squop-up situations. A 
change to this rule is not part of my current proposal, but I would like 
to get a feeling for opinion on this issue. Maybe an indicative straw 
poll at next rules meeting.

+++

Thank you for your attention - and for your good work on bringing rules 
issues to light.

**Comment by fluppeteer** [comment]
Also posting the relevant bit of my email response, for archival reasons:
<hr>
Oh, I'm fine with "we play five rounds plus a number of turns". It's "we sometimes play six whole rounds" that makes it weird to say "we play five rounds". Patrick's version of the rule works for me in terms of clarifying this, although I think that's only unambiguous if we're also clear about when turns start, however we choose to do that. (If a "turn" can be interpreted to start with the first action taken in it, we could still have six rounds; if the "turn" ends when the last action in it is completed and a new turn starts immediately after the previous one, the "blue wins squidge off, blue plays, time expires, green plays in 0" thing makes sense, because time expired during green's turn.)


Basically, I think it helps to know what we were trying to achieve with this rule, so people remember it:

"Everyone gets to play in five rounds"
=> end of CFSoW's turn ends the first turn of round 1

"Everyone gets to play five rounds' worth of shots"
=> first action of CFSoW's turn starts the first turn of round 1

"The SoW ends a turn to start round 1" (aka "everyone has five
complete turns", but only if we define "turns" as consecutive)
=> end of SoW's turn starts round 1

"The SoW gets a complete turn's worth of shots to begin round 1"
=> first action of CFSoW's turn starts the last turn before rounds

"The SoW gets a complete turn before starting round 1"
=> end of SoW's full turn starts round 1

"Everyone gets to play in five rounds" is a logical goal for me, as is "Everyone gets to play five rounds' worth of shots". "Everyone has five complete turns" seems like a less useful goal, so my objection has tended to be that "the SoW ends a turn to begin round 1" variants feel like putting the implementation before the intent. But there's a certain logic to "rewarding" the SoW with beginning rounds.

[...]
(Re.22.3.4)

I'm happy to poll the masses, but I'd fairly strongly oppose this change. If there's a squop-up just before rounds, the players can deliberately dither until time expires, and then use the squop-up
to consume most of the shots of rounds (I assume this is why 22.3.4 exists). Once time has expired, I absolutely play differently to ensure that I don't get squopped up (if I can), whereas I'm more
aggressive during timed play. With enough free winks, "just before time expires" could be five minutes, in thirty-second chunks. Given that the team with the squop-up is highly motivated to limit the number of turns remaining, I'm sure this would be used tactically; I believe the game is more interesting if the winning team is at greater risk (the same as my argument in the forthcoming WW that the risk of scrunging when trying to pot out is good for the game).

**Comment by fluppeteer** [comment]
(See a thing I just noted on the proposal to define a Turn: With the current definition, what happens if the squidge-off winner plays an illegal shot and time expires before they get the chance to replay it?)

**Comment by hatj2** [comment]
> (See a thing I just noted on the proposal to define a Turn: With the current definition, what happens if the squidge-off winner plays an illegal shot and time expires before they get the chance to replay it?)

I think this is a new issue.

**Comment by hatj2** [comment]
> As discussed by email (incompetently, in my case), I think our options are as follows. Values in the table show the round in which the shot after time expires is played.
> Edit: Added (in parentheses) which colour can control which round follows time expiring by dithering either side of time expiring, and slight paraphrase of some wording.
> 
> 
> Rule
> BTEB
> YTEB
> GTEG
> BTEG
> 
> 
> 1. "After time expires, play continues until the SoW completes a full Turn."
> -1(-)
> -1(Y)
> 0(-)
> 0(-)
> 
> 
> 2. "Play continues until the SoW completes a Turn in which no action has previously been taken."
> -1(B)
> 0(Y/B)
> 0(-)
> 0(-)
> 
> 
> 3. "Play continues until the first end of a SoW Turn after time has expired." aka "The beginning of the first Turn of the CFSoW after time has expired begins round 1."
> 0(B)
> 0(B)
> 0(-)
> 0(B)
> 
> 
> 4. "Play continues until the end of the first Turn of the CFSoW in which no action has previously been taken, which then ends the first Turn of round 1."
> 0(-)
> 0(B)
> 0(G)
> 1(B/G)
> 
> 
> 5. "The first end of a CFSoW Turn to occur after time has expired ends the first Turn of round 1."
> 0(-)
> 0(-)
> 1(G)
> 1(G)
> 
> 
> (In case 2, Yellow dithering can force Blue into ending round 0 with its next shot; Blue can avoid doing so by rushing. In case 4, Blue can force Green into round 1 by dithering, and Green can rush an action to stay in round 0 without ending Green's Turn.)
> Assumes Blue won the squidge-off, "SoW" = "squidge-off winner" and "CFSoW" is "the Colour following the squidge-off winner". I'm using my definition of "Turn" - ending when the last shot in the turn ends, and beginning immediately after the previous Turn. "Action" here means "shot, declaration of pass, or determination of inability to play" as distinct from the "thinking time" component of my general definition of a "Shot".
> For the cases, letters indicate a shot by the Colour beginning with that letter, "TE" indicates the point at which time expires".
> I believe option 2 is the current rules. I believe Patrick proposed 3. I think 5 most embodies the spirit of "everyone gets to play in five rounds at the end of the game", but I don't mind 3 if we start saying "up to six turns" rather than "at least five". Option 3 means the squidge-off winner acts to take us into round 1; it's the only case where the SoW uniquely decides when rounds happen relative to time expiring, for better or for worse (if I've got it right, which I may not have).

**Comment by hatj2** [comment]
We have to consider Patrick's wording: 

> 18. After the timed period has ended, a round limit period is played.
Play continues using the normal sequence of colours. After the turn ends
of the colour that won the squidge-off, five further rounds are played,
with each round consisting of one turn for each colour in the normal
sequence.

This corresponds to 3) in Andrew's table above. I think it clarifies the multi-turn shot issue, which I think is so rare that most players wouldn't instinctively know either way whether time expiring in the middle of SoW's turn puts them in 1 or 0 after that turns ends; I certainly have no idea at present and don't much care. Therefore I think I approve of this wording, subject to defining elsewhere what a Turn is.
